odl/contrib/datasets/README.md

```diff
@@ -21,3 +21,6 @@ Reference datasets with accompanying ODL geometries etc.
     CT data as provided by FIPS. The data is non-human and high resolution.
     * `walnut_data`
     * `lotus_root_data`
+* `images`
+
+  miscellaneous images.
```

Comment:
I guess it would be nice if you added a sub-section for `cambridge`, and in there you can also add a direct link to `http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/` so users can go there from github

Reply:
What do you mean here? How would I add a link?

Reply:
You can add it in cleartext, e.g.

```
* `images`

  Two dimensional images.
  * `cambridge`
  
    Various images as provided by the University of Cambridge, see http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/
    * `brain_phantom`
    * `resolution_phantom`
    * `building`
    * `rings`
    * `blurring_kernel`    
```